fea-rs/src/bin/compile.rs

```diff
@@ -31,10 +31,25 @@ fn run() -> Result<(), Error> {
     if !fea.exists() {
         return Err(Error::EmptyFeatureFile);
     }
-    //FIXME: some way to provide variation info from command line?
-    let compiled = Compiler::new(fea, &glyph_names)
-        .with_opts(Opts::new().make_post_table(args.post))
-        .compile()?;
+
+    let var_info = args.get_var_info().transpose()?;
+
+    let mut compiler =
+        Compiler::new(fea, &glyph_names).with_opts(Opts::new().make_post_table(args.post));
+    if let Some(var_info) = var_info.as_ref() {
+        log::info!("compiling with {} mock variation axes", var_info.axes.len());
```

Comment:
Nit: "mock" is curious terminology, if I'm the user and I told you to run with some axes to me they are real, not "mock"

Reply:
fair, what I'm trying to communicate here is that we will not be correctly compiling anything that relies on variations when called from the command line, and that we're just using these for validation.

The term 'mock' ended up here because I reused a type called `MockVariationInfo` that was initially designed for use when writing tests.